src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, ` + `chip version: ${this.version}, `
+        + `wheels: ${this.wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(number = 1) {
+      if (number < 0) {
+        return this;
+      }
+      this.coords.y += number;
+
+      return this;
```

Comment:
You can short this logic 
```suggestion
      if (number > 0) {
        this.coords.y += number;
      }
     
      return this;
```